src/sandstorm/run-bundle.c++

```diff
@@ -1719,7 +1781,8 @@ private:
             true));
       }
 
-      KJ_SYSCALL(setenv("PORT", kj::str(config.port).cStr(), true));
+      // TODO: ",".join(config.ports) rather than relying on default behavior.
```

Comment:
+1 to `strArray()`, I don't love that the default seemingly output has spaces in it.